projects/igniteui-angular/src/lib/services/overlay/utilities.ts

```diff
@@ -174,9 +178,10 @@ export class Util {
      * @hidden
```

Comment:
If the whole class is marked as `@hidden / @internal`, its members do not need to be marked separately.